errai-config/src/main/java/org/jboss/errai/config/rebind/EnvUtil.java

```diff
@@ -296,7 +296,7 @@ private static void addSerializableTypes(final Set<MetaClass> exposedClasses, fi
       MetaClassFactory
         .getAllCachedClasses()
         .stream()
-        .filter(mc -> filter.apply(mc.getFullyQualifiedName()))
+        .filter(mc -> filter.apply(mc.getFullyQualifiedName()) && !mc.isInterface() && (!mc.isAbstract() || mc.isEnum()))
```

Comment:
Not sure I understand why we need to check if the type is an enum. If `mc.isEnum()` is true, then isn't it already the case that `!mc.isAbstract()` and `!mc.isInterface()`?